common/test/db_cache/bok_choy_schema.sql

```diff
@@ -623,6 +623,20 @@ CREATE TABLE `certificates_certificategenerationcoursesetting` (
   KEY `certificates_certificategenerationcoursesetting_b4b47e7a` (`course_key`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
+DROP TABLE IF EXISTS `certificates_certificatehtmlviewconfiguration`;
```

Comment:
Agreed, but this is what we have for now. AFAICT, if this is not updated in general it just means that migrations will have to be applied, which will make the tests slower. Unfortunately in this case, I had to update the script because of the VAL profile changes. The bokchoy tests were failing on Jenkins without the change.